barretenberg/cpp/src/barretenberg/relations/translator_vm/translator_delta_range_constraint_relation.hpp

```diff
@@ -8,14 +8,14 @@ template <typename FF_> class TranslatorDeltaRangeConstraintRelationImpl {
     using FF = FF_;
 
     // 1 + polynomial degree of this relation
-    static constexpr size_t RELATION_LENGTH = 6; // degree((lagrange_last-1) * D(D - 1)(D - 2)(D - 3)) = 5
+    static constexpr size_t RELATION_LENGTH = 7; // degree((lagrange_last-1) * D(D - 1)(D - 2)(D - 3)) = 5
```

Comment:
the comment needs to be updated accordingly right?

Reply:
indeed